interactive/mechanism/species.py

```diff
@@ -7,10 +7,10 @@
 
 logging.basicConfig(format='%(asctime)s - %(message)s', level=logging.INFO)
 
-species_path = os.path.join(settings.BASE_DIR, "dashboard/static/config/camp_data/species.json")
+# species_path = os.path.join(settings.BASE_DIR, "dashboard/static/config/camp_data/species.json")
 
 # returns the full set of json objects from the species file
-def species_info():
+def species_info(species_path=os.path.join(settings.BASE_DIR, "dashboard/static/config/camp_data/species.json")):
```

Comment:
there are lots of these path constants. How would you feel about making a constants.py in each folder with the a class or some structure holding all of these constants so that if we change the location the code is easier to maintain?

Reply:
these paths constants are to still support single users -- or rather support the server still hosted on port 8000. once multi-user + rabbit mq and absolutely everything is done and old code is removed these constants will be gone